tests/test_csc.py

```diff
@@ -72,16 +72,25 @@ def basic_make_csc(self, initial_state, simulation_mode=1, config_dir=None):
             config_dir=config_dir,
         )
 
-    async def check_telemetry(self):
-        """Check the the next rotation and motors telemetry messages."""
+    async def check_telemetry(self, nskip=0):
+        """Check the the next rotation and motors telemetry messages.
+
+        Parameters
+        ----------
+        nskip : `int`
+            Number of telemetry samples to skip, for each topic.
+            Typically 0, but use 1 at a transition, e.g. starting
+            or ending a move.
+        """
         # We need some margin, slightly more than I naively expected,
         # possibly because there is some roundoff error in converting time
         # between TAI unix seconds and astropy times.
         slop = 1e-6
 
-        rotation_data = await self.remote.tel_rotation.next(
-            flush=True, timeout=STD_TIMEOUT
-        )
+        for _ in range(nskip + 1):
```

Comment:
It is useful when the variable is not used anywhere because it avoids linters complaining about unused variables.

It would be a terrible choice if the value was actually used.